Project-I-Social-Card/social-card/src/components/CardComponents/Card.css

```
@@ -1 +1,28 @@
 /* All styles for your Card Components live here */
+
+.card-container {
+    border: solid gray 1px;
+    margin-left: 96px;
+    width: 650px;
+    height: 485px;
```

I went through and changed all the heights to auto. It cleaned up the content that was spilling out.

```suggestion
    height: auto;
```

---

Other than that, your social card looks great!